source/commands/gitops/create/github.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Text } from 'ink';
+import zod from 'zod';
+import { option } from 'pastel';
+import SelectProject from '../../../components/gitops/SelectProject.js';
+import PolicyName from '../../../components/gitops/PolicyName.js';
+import SSHKey from '../../../components/gitops/SSHKey.js';
+import BranchName from '../../../components/gitops/BranchName.js';
+import Activate from '../../../components/gitops/Activate.js';
+import {
+	KEYSTORE_PERMIT_SERVICE_NAME,
+	DEFAULT_PERMIT_KEYSTORE_ACCOUNT,
+} from '../../../config.js';
+import * as keytar from 'keytar';
+
+type GitConfig = {
+	url: string;
+	main_branch_name: string;
```

Comment:
Please keep the camelCase naming convention on all the code variable names (you can create separate one for API use only)